src/shared/components/login/socialLogin.js

```diff
@@ -0,0 +1,172 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import config from 'config/environment';
+import Section from 'shared/components/section/section';
+import Form from 'shared/components/form/form';
+import FormZipCode from 'shared/components/form/formZipCode/formZipCode';
+import FormPassword from 'shared/components/form/formPassword/formPassword';
+import FormButton from 'shared/components/form/formButton/formButton';
+import styles from 'scenes/home/informationForm/informationForm.css';
+import _ from 'lodash';
+import * as CookieHelpers from '../../utils/cookieHelper';
+
+class SocialLogin extends Component {
+  state = {
+    zip: '',
+    password: '',
+    error: false,
+    isLoading: false
+  }
+
+  componentWillUnmount() {
+    this.onExit();
+  }
+
+  onZipChange = (value, valid) => {
+    this.setState({ zip: value, zipValid: valid });
```

Comment:
Big no-no to set state on variables not initially in the state object.

Be sure to map `zipValid` and `passwordValid` to the state object in this component. I believe `false` will be a good initial value, but I'm not certain.

Reply:
done.